### PR TITLE
update test_git.py

### PR DESCRIPTION
**Pull request by kshitij3199**
update method, test_raw_to_enrich_projects in test_git.py
Signed-off-by: Kshitij Gupta <kgupta3199@gmail.com>

### CODE REVIEW

**Comment by vchrombie** [comment]
Hi @kshitij3199 
I think you missed the sign-off on the commit. You can amend the commit and try again.

Reference: https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md#dco-and-sign-off-for-contributions

**Comment by kshitij3199** [comment]
@vchrombie  I don't why this is  happening, but I have written sign off, do we have to do it explicitly my terminal

**Comment by vchrombie** [comment]
Hi @kshitij3199 

You need not add it manually, whenever you are committing, add `-s` before `-m` to add a sign-off.

For now, as the problem is only with the last commit, try this
```
git commit --amend --no-edit --signoff
```

It should work. Let me know if you have any issues. :slightly_smiling_face:

**Comment by vchrombie** [comment]
> @vchrombie I don't know why this is happening, but I have written sign off, do we have to do it explicitly my terminal

Oh! :thinking: 

> Signed-off-by: Kshitij Gupta [kgupta3199@gmail.com](mailto:kgupta3199@gmail.com)

Ya, I can see that here too. Anyways try again. It should not be a problem.

**Comment by vchrombie** [comment]
> For now, as the problem is only with the last commit, try this
> 
> ```
> git commit --amend --no-edit --signoff
> ```

and then force push the branch.

**Comment by animeshk08** [comment]
Hi, @kshitij3199 You can also have a look at your travis-ci logs. It is failing due to flake 8 checks which are easy to get rid of :)
https://travis-ci.org/github/chaoss/grimoirelab-elk/jobs/662481920?utm_medium=notification&utm_source=github_status

File: tests/test_git.py [hunk 1]
```diff
@@ -154,6 +154,27 @@ def test_raw_to_enrich_projects(self):
...
+                 self.assertIn('utc_author_date_weekday', source)
+                 self.assertIn('utc_author_date_hour', source)
+                 self.assertIn('author_uuid', source)
+                 self.assertIn('author_domain', source)
+                 self.assertIn('author_name', source)
+                 self.assertIn('commit_date', source)
+                 self.assertIn('commit_date_weekday', source)
+                 self.assertIn('commit_date_hour', source)
+                 self.assertIn('utc_commit_date_weekday', source)
+                 self.assertIn('utc_commit_date_hour', source)
```
Comment (valeriocos) [inline tests/test_git.py]: This test is used to check that the attributes `project` and `project_1` are in the enriched indexes. Please add some assertions like the ones below, thanks:
- self.assertIn('project', source)
- self.assertIn('project_1, source)
- self.assertEqual(source['project'], 'Main)
- self.assertEqual(source['project_1'], Main)

**Review by valeriocos — COMMENTED** [review]
Thank you @kshitij3199 for the PR! (Thank you @vchrombie @animeshk08 to take a look at the PR)

- Please check the comment of @vchrombie to sign your commit:
`git commit --amend --signoff` should also work 

- Please check the comment of @animeshk08 to fix the flake8 errrors:
https://travis-ci.org/github/chaoss/grimoirelab-elk/jobs/662481920?utm_medium=notification&utm_source=github_status

- Please check the review comments to make sure that the project's attributes are present in the enriched index

**Comment by kshitij3199** [comment]
Thank you @valeriocos @vchrombie @animeshk08 
 I will resolve all the issues and update the PR as soon as I reach home , ( I will be able to update it by Tuesday , due to COVID19 our university has asked us to go home , sorry for delay)

**Comment by valeriocos** [comment]
@kshitij3199 I understand that this PR can be closed

**Pull request closed without merging**